src/components/popover/tab/keypress/keypress-tab.component.html

```diff
@@ -26,4 +26,10 @@
     <select2 #longPressSelect [data]="longPressGroups" [selectedValue]="selectedLongPressIndex.toString()" (valueChanged)="onLongpressChange($event)"
         [width]="140"></select2>
     <icon name="question-circle" title="This action happens when the key is being held along with another key."></icon>
+</div>
+
+<div class="disabled-state--text">
```

Comment:
This is currently hidden, but we will toggle visibility when logic is added later. This text is shown if user select key as layer switcher key.